koku/api/settings/tag_management.py

```diff
@@ -1,6 +1,18 @@
 #
-# Copyright 2021 Red Hat Inc.
-# SPDX-License-Identifier: Apache-2.0
+# Copyright 2020 Red Hat, Inc.
```

Comment:
Just as a heads up @ddonahue007, we switched our project license a while back from GPL to Apache. So we actually need the license header from before: 
```
#
# Copyright 2021 Red Hat Inc.
# SPDX-License-Identifier: Apache-2.0
#
```

Reply:
Thanks